dags/overwatch.py

```diff
@@ -0,0 +1,53 @@
+"""
+Overwatch
+Runs daily at 0700 UTC
+
+Source code is [overwatch repository](https://github.com/mozilla/overwatch/).
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models import Variable
+from operators.gcp_container_operator import GKEPodOperator
+
+docs = """\
+This DAG is executes the Overwatch monitoring system.
+All alerts related to this DAG can be ignored.
+
+(for more info on dim see: https://github.com/mozilla/overwatch-mvp)
+"""
+
+
+default_args = {
+    "owner": "gleonard@mozilla.com",
+    "email": [
+        "gleonard@mozilla.com",
+        "wichan@mozilla.com",
+    ],
+    "start_date": datetime(2022, 11, 1),
```

Comment:
This will run on entire month's worth of data

Reply:
Not unless `catchup=True` is passed into the DAG. Here's an example:
https://github.com/mozilla/telemetry-airflow/blob/main/dags/data_monitoring.py#L51

Reply:
I just quickly researched this. So apparently in Airflow the default is `True`. However, checking our config it appears we are defaulting to `False`. You can check our Airflow config by going to WTMO -> Admin -> Configuration and searching for `catchup_by_default`

Reply:
I've updated the DAG to start `2022-11-23` with `catchup=True` to capture the changes in metrics over the holiday in the US.